sdk/@launchdarkly/observability-dotnet/test/LaunchDarkly.Observability.Tests/log-test-scenarios.json

```diff
@@ -0,0 +1,392 @@
+[
+	{
+		"description": "Should match a log based on severity",
+		"samplingConfig": {
+			"logs": [
+				{
+					"severityText": {
+						"matchValue": "Error"
```

Comment:
The logging setup in dotnet requires actual log levels, and those log levels have specific string representations, so this is modified to use those levels.